src/web/mjs/connectors/Toonkor.mjs

```diff
@@ -7,17 +7,32 @@ export default class Toonkor extends GnuBoard5BootstrapBasic2 {
         super.id = 'toonkor';
         super.label = 'Toonkor';
         this.tags = ['webtoon', 'korean'];
-        this.url = 'https://tkor.zone';
+        this.url = 'https://t.me/s/new_toonkor';
+        this.hostRegexp = /https?:\/\/(?:toonkor|tkor)([\d]+)?\.[a-z]+/;
     }
 
     canHandleURI(uri) {
-        return /https?:\/\/(?:toonkor|tkor)\.[a-z]+/.test(uri.origin);
+        return this.hostRegexp.test(uri.origin);
     }
 
     async _initializeConnector() {
-        let uri = new URL(await this._twitter.getProfileURL('1202224761771741184') || this.url);
-        let request = new Request(uri.href, this.requestOptions);
-        this.url = await Engine.Request.fetchUI(request, `window.location.origin`);
+        const script = `
+            new Promise((resolve, reject) => {
+                setTimeout(() => {
+                	  //fetch telegram messages with links, reverse it because last one is more recent one, and get the first matching our regex
+                    const tkLinks = [...document.querySelectorAll('section.tgme_channel_history div.tgme_widget_message_wrap .tgme_widget_message_text a ')].reverse();
+                    for (const link of tkLinks) {
+                        if ( ${this.hostRegexp}.test(link.href)) {
```

Comment:
Interesting, wouldn't expect an RegExp to correctly work when interpolated inside a string